server/datastore/mysql/software_titles_test.go

```diff
@@ -865,6 +867,19 @@ func testTeamFilterSoftwareTitles(t *testing.T, ds *Datastore) {
 	require.NoError(t, err)
 	require.Len(t, titles, 1)
 	require.Equal(t, "vpp3", titles[0].Name)
+
+	// Testing with Platform filter
+	titles, count, _, err = ds.ListSoftwareTitles(
```

Comment:
Should probably check this or not assign it, the linter is upset with it

Reply:
done!